### PR TITLE
UI: update Transcription (and update local dev app name)

### LINKED ISSUE #897: Use IPA in the conbench transcription
A follow on from #879, uses IPA for transcription now (so we're not connected to a particular English variety as a referent)

### PR DESCRIPTION
**Pull request by jonkeane**
Resolves #897

IPA transcriptions are broadly preferred to the English-based orthographic versions like we had (and we get to use fun characters like `ʧ` 😛 )

Also changes the name for local deployments to something slightly more obvious that that is what it is.

### CODE REVIEW

File: conbench/tests/api/_expected_docs.py [hunk 1]
```diff
@@ -1329,7 +1329,7 @@
             },
         },
     },
-    "info": {"title": "conbench-for-pytest", "version": "1.0.0"},
+    "info": {"title": "local-dev-conbench", "version": "1.0.0"},
```
Comment (jonkeane) [inline conbench/tests/api/_expected_docs.py:1332]: Hope I'm not stepping on any toes here. I was confused a little bit the first few times I saw this (locally + also in screenshots) that this was suggesting conbench is some sort of pytest extension or connector. Hopefully this makes it super duper clear what's up when we see it (e.g. this is a local deployment!)

**Comment by austin3dickey** [comment]
tests are failing due to #895

**Comment by jgehrcke** [comment]
Cool! Can you change the PR title to something like "UI: update Transcription (and update local dev app name)" so that we get a nice commit msg after merge?

**Review by jgehrcke — CHANGES_REQUESTED** [review]
Great, PR title adjustment would be nice, otherwise :ship:

File: conbench/tests/api/_expected_docs.py (hunk 1 above)
Comment (jgehrcke) [reply conbench/tests/api/_expected_docs.py:1332]: :) this is good as long as it's not "Conbench"

**Comment by jgehrcke** [comment]
I think the `ci/ui-screenshots` failure might also be because of https://github.com/conbench/conbench/issues/895. I didn't look into it yet. Someone should.

**Review by jgehrcke — APPROVED** [review]
Oh "Changes requested" is too harsh lol, I think this can only be unlocked by pushing a commit.

**Comment by austin3dickey** [comment]
>I think the ci/ui-screenshots failure might also be because of https://github.com/conbench/conbench/issues/895. I didn't look into it yet. Someone should.

Yep! The container logs show many "hourly rate limit exceeded" messages

File: conbench/templates/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {% block app_content %}
 
 
-<h1 class="display-3 text-center" style="margin-bottom: 40px">/kənˈben(t)SH/</h1>
+<h1 class="display-3 text-center" style="margin-bottom: 40px">/ˈkɒnbɛnʧ/</h1>
```
Comment (jgehrcke) [inline conbench/templates/index.html:6]: An interesting question might be if all systems we care about can display those. Probably!

**Comment by jgehrcke** [comment]
> IPA transcriptions are broadly preferred to the English-based orthographic versions like we had (and we get to use fun characters like ʧ stuck_out_tongue )

Oh and @jonkeane it seems like you can teach us a lot about the underlying concepts here :) Some see IPA and think of beer :beers:

File: conbench/templates/index.html (hunk 2 above)
Comment (austin3dickey) [reply conbench/templates/index.html:6]: I did notice that our screenshot tool cannot. I don't think that's a big deal though. It also uses monospaced font, so I don't pay much attention to its visual properties.
frontpage

Comment (jgehrcke) [reply conbench/templates/index.html:6]: > I did notice that our screenshot tool cannot. 

Interesting! We can look into that. It's probably a widely discussed topic -- font support in headless chrome.

**Comment by jgehrcke** [comment]
> Can you change the PR title to

quickly took liberty to do that myself & merge :).

**Comment by jonkeane** [comment]
Thanks for pushing and merging 🙏

**Comment by jonkeane** [comment]
https://github.com/apache/arrow/pull/13207#discussion_r881864500 has some discussion around transcription, but TL;DR IPA is basically the one and only way to transcribe spoken languages that isn't connected to a specific language + orthograph

**Pull request merged**